EDI/Overview/03_Processes.md

```diff
@@ -0,0 +1,3 @@
+# Processes overview
+
+![Process](../../Assets/Screenshots/EDI/Overview/Process.png "[Process]")
```

Comment:
Ist "Workflow-triggered export" gemeint? (statt Workflow - Trigger export)

Reply:
ne der Workflow triggered den Export
Docu processes ist auch in den anderen Schnittstellenbeschreibungen nicht drin